src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,44 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  constructor(name, health = Animal.defaultHealth) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  static kill(victim) {
+    Animal.alive = Animal.alive.filter(animal => animal !== victim);
+  }
 }
 
+Animal.alive = [];
+Animal.defaultHealth = 100;
```

Comment:
this is a proper way to mutate Animal class by adding static prop, but not actually needed. See the above.